PlainTasksDates.py

```diff
@@ -548,19 +548,19 @@ def extract_tag(self, point):
             tag under cursor (i.e. point)
         '''
         start = end = point
-        tag_pattern = r'(?<=\s)(\@[^\(\) ,\.]+)([\w\d\.\(\)\-!? :\+]*)'
-        line = self.view.line(point)
-        matches = re.finditer(tag_pattern, self.view.substr(line))
-        for match in matches:
-            m_start = line.a + match.start(1)
-            m_end   = line.a + match.end(2)
-            if m_start <= point <= m_end:
-                start = line.a + match.start(2)
-                end   = m_end
-                break
-        else:
-            match = None
-        tag = match.group(0) if match else ''
+        tag = ''
+        if 'tag.todo' in self.view.scope_name(point):
+            reg = self.view.extract_scope(point)
```

Comment:
I'll be taking a break for the exam period but I'll finish up a few improvements when I get time. I've  got a in-dev copy that works for just the settings I want but I'll need to polish it so it works for the master release